remoteworkitem/trackerquery_repository_blackbox_test.go

```diff
@@ -42,7 +40,7 @@ func (s *trackerQueryRepoBlackBoxTest) TestFailDeleteZeroID() {
 		s.Ctx,
 		"project = ARQ AND text ~ 'arquillian'",
 		"15 * * * * *",
-		tr.ID, space.SystemSpace)
+		testFxt.Tracker[0].ID, space.SystemSpace)
```

Comment:
When we talked I thought the purpose was to get rid of the `space.SystemSpace` and some other dependencies on implicitly available objects and not to introduce a new recipe function.

And correct me if I'm wrong, but a tracker only contains these two fields which you almost always want to customize anyway, don't you?

```go
	// URL of the tracker
	URL string
	// Type of the tracker (jira, github, bugzilla, trello etc.)
	Type string
```

Reply:
After talking to you, let's keep the Tracker recipe. :)